src/main/java/org/pesmypetcare/webservice/controller/petmanager/WeightRestController.java

```diff
@@ -0,0 +1,115 @@
+package org.pesmypetcare.webservice.controller.petmanager;
+
+
+import org.pesmypetcare.webservice.service.WeightService;
+import org.pesmypetcare.webservice.entity.WeightEntity;
+import org.pesmypetcare.webservice.error.DatabaseAccessException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/weight")
+public class WeightRestController {
+    @Autowired
+    private WeightService weightService;
+
+    /**
+     * Creates a weight of the pet on the database.
+     * @param owner Username of the owner of the pet
+     * @param petName Name of the pet
+     * @param weightEntity The parameters of weight
+     */
+    @PostMapping("/{owner}/{petName}/{petDate}")
+    public void createWeight(@PathVariable String owner, @PathVariable String petName, @PathVariable String petDate,
+                             @RequestBody WeightEntity weightEntity) {
+        weightService.createWeight(owner, petName, petDate, weightEntity);
+    }
+    /**
```

Comment:
All methods should be separated by one empty line

Reply:
Solved!